docs/docusaurus/docs/core/introduction/try_gx.md

```diff
@@ -1,7 +1,153 @@
 ---
 title: Try Great Expectations
 ---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-import InProgress from '../_core_components/_in_progress.md'
+import GxData from '../_core_components/_data.jsx';
+import PrereqPythonInstalled from '../_core_components/prerequisites/_python_installation.md';
+import ReleaseVersionBox from '../../components/versions/_gx_version_code_box.mdx'
+import GxCloudAdvert from '/static/docs/_static_components/_gx_cloud_advert.md'
+
+Follow this guide to install {GxData.product_name}, connect to sample data, build your first Expectation, validate the sample data, and review the Validation Results.  This is an ideal place to start if you're new to GX and would like to experiment with some basic features of GX to see what it has to offer.
+
+## Prerequisites
+
+- <PrereqPythonInstalled/>
+
+## Setup
+
+{GxData.product_name} is a Python library and can be installed using Python's included `pip` tool.
+
+More comprehensive guidance on setting up a Python environment, installing GX, and installing additional dependencies for working with specific data formats and storage environments is available in the [Set up a GX environment](/core/installation_and_setup/install_gx.md) section of the docs. 
+
+<Tabs>
+
+<TabItem value="procedure" label="Procedure">
+
+1. Install the GX 1.0 library.
+
+   The following command will install GX in your Python environment:
+
+   ```bash title="Terminal input"
+   pip install great_expectations
+   ```
+
+   Since this guide provides sample `.csv` data to work with, no additional dependencies need installation.
+
+2. Verify that GX installed successfully.
+
+   Once the {GxData.product_name} library has been installed, you can verify that you are using the correct version with:
+
+   ```bash title="Terminal input"
+   great_expectations --version
+   ```
+
+   The output you receive if GX was successfully installed will be:
+
+   <ReleaseVersionBox/>
+
+</TabItem>
+
+</Tabs>
+
+## Try GX
+
+<Tabs>
+
+<TabItem value="procedure" label="Procedure">
+
+1. Import the required modules from the {GxData.product_name} library.
+
+   From a Python interpreter, IDE, or script run the following import statements:
+
+   ```python title="Python input" name="docs/docusaurus/docs/core/introduction/try_gx.py imports"
+   ```
+   
+   The `great_expectations` module is the root of the GX library and contains shortcuts and convenience methods for starting a GX project in a Python session.
+
+   The `expectations` module contains all the Expectation classes that are provided by the GX library.
+
+2. Create a temporary Data Context and connect to the provided sample data.
+
+   A Data Context can be thought of as a GX project.  In Python, it provides the API for interacting with many common GX objects.
```

Comment:
```suggestion
   In Python, a Data Context provides the API for interacting with GX objects.
```

Reply:
Slightly tweaked the suggestion.  I think "most" is an important distinction here since we're having them import the `expectations` module to get to Expectation classes, rather than doing so through the Data Context.